specs/altair/sync-protocol.md

```diff
@@ -126,11 +126,11 @@ def validate_light_client_update(snapshot: LightClientSnapshot,
     else:
         signed_header = update.finality_header
         assert is_valid_merkle_branch(
-            leaf=hash_tree_root(update.header),
+            leaf=hash_tree_root(update.finality_header),
```

Comment:
`update.finality_header` is an expansion of `state.finalized_checkpoint.root`, which corresponds to `FINALIZED_ROOT_INDEX`.